src/layouts/EditHomepage/EditHomepage.jsx

```diff
@@ -139,6 +157,7 @@ const EditHomepage = ({ match }) => {
     image: "",
     notification: "",
     sections: [],
+    announcements: {},
```

Comment:
Should `announcements` be inside `sections` instead? It is on the same level as `infopic` and `infobar`.

Actually I quite confused, the `onFieldChange` part puts the announcements inside `sections`, which is the correct behaviour. Perhaps this part is not needed?

Reply:
hey good catch, you are right this is not needed, removed it here 342581a2cfa4620a056b0334bf5187c31f9b198b